source/containers.tex

```diff
@@ -5551,8 +5554,11 @@
 \remarks
 Causes reallocation if the new size is greater than the old capacity.
 Reallocation invalidates all the references, pointers, and iterators
-referring to the elements in the sequence.
-If no reallocation happens, all the iterators and references before the insertion point remain valid.
+referring to elements in the sequence, as well as the past-the-end iterator.
+If no reallocation happens, all references, pointers, and iterators
```

Comment:
This bit is normative duplication.

Reply:
Yes, but it's part of a sentence where the second part is *not* duplication. Can you think of a way to re-word it?

Reply:
```tex
Invalidates all references, pointers, and iterators from the insertion point forward, including the
past-the-end iterator. If the new size is greater than the old capacity, causes reallocation which
additionally invalidates all the references, pointers, and iterators referring to elements in the
sequence which precede the insertion point. \begin{note} When no reallocation occurs,
references, pointers, and iterators before the insertion point remain valid. \end{note} If an
exception is thrown ...
```

Reply:
I guess I'm concerned about the potential for misreading "from the insertion point" as exclusive rather than inclusive. (Also, "forward" might better be "onward"? I guess that doesn't matter as it's hard to misread.)
My legalistic eye doesn't like how Casey's wording drops the "referring to elements" part, or uses "including" instead of "as well as"... but nonetheless, I would be fine with Casey's wording making it into the standard.

Reply:
By all means, suggestive alternatives. We do WG21 and ourselves a favor by banging out wording upon which we can agree before asking LWG to rubber stamp it.